automatilib/i_dot_ai_utils.py

```diff
@@ -1 +1,13 @@
 """Main module."""
+from django.http import HttpResponseNotAllowed
+
+
+class MethodDispatcher:
```

Comment:
lets avoid meta programming, e.g. `getattr` unless absolutely unavoidable, id much rather we just type it all out manually especially for a common package